src/WebJobs.Script/Host/FunctionMetadataManager.cs

```diff
@@ -215,7 +216,7 @@ private void LoadCustomProviderFunctions(List<FunctionMetadata> functionMetadata
 
         private void AddMetadataFromCustomProviders(IEnumerable<IFunctionProvider> functionProviders, List<FunctionMetadata> functionMetadataList)
         {
-            _logger.FunctionMetadataProviderParsingFunctions();
+            _logger.FunctionMetadataProviderParsingFunctions(nameof(FunctionMetadataManager));
```

Comment:
I think you meant to use _customProviderLog here as well?

Reply:
Also, for all these Host/Worker/Custom const fields, let's name them all _metadataProviderName in each of their respective classes

Reply:
Changed!